src/commons/error.rs

```diff
@@ -755,7 +755,7 @@ impl Error {
 
             Error::CaRepoIssue(ca, err) => ErrorResponse::new("ca-repo-issue", &self).with_ca(ca).with_cause(err),
 
-            Error::CaRepoResponseInvalidXml(ca, err) => ErrorResponse::new("ca-repo-response-invalid-xml", &self)
+            Error::CaRepoResponseInvalid(ca, err) => ErrorResponse::new("ca-repo-response-invalid-xml", &self)
```

Comment:
A minor point perhaps, but I note that you removed the XML suffix from the enum member name but not from the language key. This seems inconsistent and a bit confusing.

Reply:
yeah, this is because of the (far?) off chance that someone might parse the error code when using the api so I thought it better not to change it.